File: problems/3264-final-array-state-after-k-multiplication-operations-i/solution.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        for _ in range(k):
            min_val = min(nums)
            min_index = nums.index(min_val)
            nums[min_index] *= multiplier
        return nums

File: problems/3264-final-array-state-after-k-multiplication-operations-i/solution.cpp
class Solution {
public:
    vector<int> getFinalState(vector<int>& nums, int k, int multiplier) {
        for (int i = 0; i < k; ++i) {
            int minVal = nums[0];
            int minIndex = 0;
            for (int j = 1; j < nums.size(); ++j) {
                if (nums[j] < minVal) {
                    minVal = nums[j];
                    minIndex = j;
                }
            }
            nums[minIndex] *= multiplier;
        }
        return nums;
    }
};